test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/EndpointSharedConfigTest.java

```diff
@@ -76,6 +76,15 @@ public void resolvesCorrectEndpoint() {
                                   .append("  endpoint_url = ").append(testCase.serviceProfileSetting).append("\n");
             }
 
+            if (testCase.serviceSectionProfileSetting != null) {
+                if (!profileFileContent.toString().contains("services = dev")) {
```

Comment:
Why is this if necessary?

Reply:
It's not. I was running into an issue with duplication, but it was an unrelated issue. Just an oversight that is safe to remove.